ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -211,21 +246,20 @@ echo "SUB_MANIFEST : ${SUB_MANIFEST} "
 SUB=$(oc create -f - -o jsonpath='{.metadata.name}' <<< "${SUB_MANIFEST}" )
 
 echo "Subscription name is \"$SUB\""
-echo "Waiting for installPlan to be created"
-# store subscription name and install namespace to shared directory for upgrade step
-echo "${OO_INSTALL_NAMESPACE}" > "${SHARED_DIR}"/oo-install-namespace
-echo "${SUB}" > "${SHARED_DIR}"/oo-subscription
-
-FOUND_INSTALLPLAN=false
-# wait up to 5 minutes for CSV installPlan to appear
-for _ in $(seq 1 60); do
-    INSTALL_PLAN=$(oc -n "$OO_INSTALL_NAMESPACE" get subscription "$SUB" -o jsonpath='{.status.installplan.name}' || true)
-    if [[ -n "$INSTALL_PLAN" ]]; then
-      oc -n "$OO_INSTALL_NAMESPACE" patch installPlan "${INSTALL_PLAN}" --type merge --patch '{"spec":{"approved":true}}'
-      FOUND_INSTALLPLAN=true
-      break
-    fi
-    sleep 5
+
+wait_for_installplan
+
+retry_attempts=2
```

Comment:
wdyt about upping this to 3 attempts?

Reply:
I was debating between 2 and 3 attempts. In the worst case scenario waiting for InstallPlan could take 5 minutes. 
So when we first run the script we could wait 5 minutes, and then if we have 3 retry attempts it would be 3*5=15 minutes, so total of 20 minutes. I thought it is too much and chose 2 attempts which in the worst case would take us 15 minutes.
But I could change it to 3 if you think it would worth it.

Reply:
I agree with your assessment, retrying twice should be good enough.